R/htmlwidgets.R

```diff
@@ -130,7 +130,7 @@ widget_data <- function(x, id, ...){
   # anywhere in the data/metadata of a widget--you will get a syntax error
   # instead of a properly rendered widget.
   tags$script(type="application/json", `data-for` = id,
-    HTML(gsub("<", "\\\\u003c", toJSON(createPayload(x))))
+    HTML(gsub("</script>", "\\\\u003c/script>", toJSON(createPayload(x))))
```

Comment:
I'd set `ignore.case=TRUE` and match on `</script`, not `</script>`. (Test case: `</SCRIPT >`